packages/custom-storage/src/sql/SQLStorage.ts

```diff
@@ -100,8 +100,8 @@ export class SQLStorage {
     return getVisualization(this._tableName, this._datasetsTableName, this._datasetsVisTableName, id, this._sql);
   }
 
-  public async getDatasetData(name: string): Promise<Dataset> {
-    return getDatasetData(name, this._sql);
+  public async getDatasetData(name: string, tablename: string): Promise<Dataset> {
```

Comment:
Given that name was used as the table name previously, why don't we change the parameter name?

I don't get how we are stripping the prefix from table's name 🤔

Reply:
It's not really needed to split table's name, we are getting both dataset name and table name when getting dataset info; so we just pass it as a parameter to keep it simple, as it is done in `custom-storage/src/sql/utils.ts@78`.